src/proof/proof_manager.h

```diff
@@ -69,15 +69,17 @@ class TheoryProof;
 class UFProof;
 class ArithProof;
 class ArrayProof;
-class BitVectorProof;
+
+namespace proof {
```

Comment:
Maybe it is better to call this namespace `bvproof` or something more indicative?